src/documents/IoCAPI.md

```diff
@@ -131,20 +131,36 @@ Holds the global root component for the global instantiator's component tree.
 
 ## Nexus API methods
 
+These API methods are grouped together here since they were developed alongside the need to support the
+GPII [Nexus](https://wiki.gpii.net/w/The_Nexus), but they are of interest to others who are building frameworks
+or authoring systems layered on Infusion's component tree substrate. They allow for the construction and destruction
+of arbitrary components at arbitrary paths in the (single, global) component tree managed by this Infusion instance.
+
 ### fluid.construct(path, options[, instantiator])
 
 Construct a component with the supplied options at the specified path in the component tree. The parent path of the
 location must already be a component.
 
-* `path {String|Array of String}` Path where the new component is to be constructed, represented as a string or array of
-  segments
+* `path {String|Array of String}` Path where the new component is to be constructed, represented as a string or array
+  of segments
 * `options {Object}` Top-level options supplied to the component - must at the very least include a field `type` holding
   the component's type. Note that these are expressed in the future-compatible
-  [post-FLUID-5750](https://issues.fluidproject.org/browse/FLUID-5750) format with `type` alongside the component's
-  options rather than at a higher nested level as is currently required in local configuration supplied as
-  [subcomponents](SubcomponentDeclaration.md).
+  [post-FLUID-5750](https://issues.fluidproject.org/browse/FLUID-5750) format with `type` alongside the component's options
+  rather than at  a higher nested level as is currently required in local configuration supplied
+  as [subcomponents](SubcomponentDeclaration.md).
 * `instantiator {Instantiator}` [optional] The instantiator holding the component to be created - if blank, the
   [global instantiator](#fluidglobalinstantiator) will be used
+* Returns: `{Component}` The constructed component
+
+### fluid.constructChild(parent, memberName, options)
+
+Constructs a subcomponent as a child of an existing component, via a call to `fluid.construct`. Note that if
+a component already exists with the member name `memberName`, it will first be destroyed.
```

Comment:
I removed the "Note" portion. If it is a note, maybe we should use the note styling.

```suggestion
Constructs a subcomponent as a child of an existing component, via a call to `fluid.construct`. If
a component already exists with the member name `memberName`, it will first be destroyed.
```